### PR TITLE
feature/mdr-1-forgot-pw-redesign

### PR DESCRIPTION
**Pull request by roxannealbert**
**What's changed:**

- Updated card title to include mdr logo
- stepper color 
- Continue button changed from 'Continue' to 'Confirm Email' and color when email is inputted
- Added 'Forgot your Password?'
- Added 'Back to Login'

**Future updates:**

- responsiveness for other screen sizes (currently for web only)
- dynamic recolor of steppers, buttons, etc.

More details here: https://github.com/orgs/Modern-Resolve-Developers/projects/4/views/6?pane=issue&itemId=27316101

### CODE REVIEW

**Review by bamsepexe — APPROVED** [review]

**Review by acostales08 — APPROVED** [review]

**Review by reanderagulto — APPROVED** [review]
Done reviewing. All goods on my end @roxannealbert @JMSevilla

File: src/components/ForgotPassword/forms/BottomButtonGroup.tsx [hunk 1]
```diff
@@ -12,18 +12,20 @@ export type BottomButtonGroupProps = {
   disableBtn?: boolean;
   onresend?: () => void
   countdown?: number
+  backtoLogin?: string;
 };
 
 export const BottomButtonGroup: React.FC<BottomButtonGroupProps> = ({
-  continueButtonLabel = "Continue",
+  continueButtonLabel = "Confirm Email",
```
Comment (JMSevilla) [inline src/components/ForgotPassword/forms/BottomButtonGroup.tsx:19]: Kindly revert this changes back to "Continue" , this props can be change when you reuse BottomButtonGroup (e.g <BottomButtonGroup continueButtonLabel="Confirm Email" />

File: src/components/ForgotPassword/forms/BottomButtonGroup.tsx [hunk 2]
```diff
@@ -58,17 +60,26 @@ export const BottomButtonGroup: React.FC<BottomButtonGroupProps> = ({
...
-          variant="outlined"
+          variant="contained"
           fullWidth
           disabled={disabledContinue}
           onClick={handleContinue}
         >
           {continueButtonLabel}
         </Button>
       </Grid>
+      <Grid item xs={8} display="flex" justifyContent="center">
```
Comment (JMSevilla) [inline src/components/ForgotPassword/forms/BottomButtonGroup.tsx:73]: I think we can use the button below under !hideBack , this code of yours will appear on all stepper implementation we have. Maybe just transfer this changes to the Button below instead of creating new one.

File: src/components/Stepper/MuiStepper/MuiStepper.tsx [hunk 3]
```diff
@@ -11,14 +11,15 @@ import {
   
  
   
-  const OnboardingStepIcon: React.FC<StepIconProps> = ({ active, completed }) => {
+  const OnboardingStepIcon: React.FC<StepIconProps & { paletteColors: any }> = ({ active, completed, paletteColors }) => {
```
Comment (JMSevilla) [inline src/components/Stepper/MuiStepper/MuiStepper.tsx:14]: We've talked about this but it seems like this paletteColors can't use dynamically based on your code here we just added a new type but the usage is not reachable outside of our stepper component. You can remove this paletteColors and just drop the required colors based on wireframe. 

Once changed : this will apply to all our stepper component already implemented.

**Review by JMSevilla — CHANGES_REQUESTED** [review]

**Review by JMSevilla — APPROVED** [review]
